test/zebras/test_period.py

```diff
@@ -115,3 +115,49 @@ def test_periods_feature(freq):
     np.testing.assert_array_equal(pr.start_time.dayofweek, ps.dayofweek)
     np.testing.assert_array_equal(pr.start_time.dayofyear, ps.dayofyear)
     np.testing.assert_array_equal(pr.start_time.isocalendar().week, ps.week)
+
+
+@pytest.mark.parametrize(
+    "data, freq, result",
+    [
+        (
+            "2023-03-19",
+            "W",
+            np.array(
+                ["2023-03-13", "2023-03-20", "2023-03-27", "2023-04-03"]
+            ).astype(np.datetime64),
```

Comment:
Can we delay the use of `np.array` to the function body. That would make this easier to read.